index.js

```diff
@@ -1547,7 +1552,7 @@
   //. Left('Cannot divide by zero')
   //. ```
   function Left(x) {
-    return new Either(sentinel, 'Left', x);
+    return new _Either('Left', x);
```

Comment:
That's elegant! I've pushed a commit which includes this change and some straightforward test cases. :)